keras_rcnn/layers/object_detection/object_proposal.py

```diff
@@ -0,0 +1,59 @@
+import keras.backend
```

Comment:
It provides a private namespace so we can keep modules short and simple. Users would import `ObjectProposal` from `keras_rcnn.layers.object_detection`.